src/main/user-impl/java/com/mysql/cj/jdbc/ha/ca/plugins/DefaultMonitorService.java

```diff
@@ -109,7 +109,6 @@ public void stopMonitoring(MonitorConnectionContext context) {
     final String node = this.threadContainer.getNode(context.getNodeKeys());
 
     if (node == null) {
-      log.logWarn(Messages.getString("DefaultMonitorService.InvalidContext"));
```

Comment:
Any reason to remove a log message?

Reply:
If `node` is null it means we are trying to remove a resource that has already been removed, I think we can just silently ignore and return in this case.